src/components/admin/lib/constants.js

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
That's no longer the description is it though? Or should we leave it like that until other use cases? Just find it not working with the name of the file